src/releaseNotes.ts

```diff
@@ -176,7 +176,7 @@ export class ReleaseNotes {
         mergeCommitSha: '',
         author: commit.author || '',
         repoName: '',
-        labels: [],
+        labels: new Set(),
```

Comment:
@invakid404 could you please elaborate why the change to `Set` would be needed?

Reply:
It's nothing too important, but it's generally better to store labels in a set to ensure we don't store duplicates and speed up lookup time when categorizing the PRs.

Reply:
Fair point. Agree, `Set` is better in this case then :)